transport/src/test/java/io/netty/channel/AbstractChannelTest.java

```diff
@@ -87,6 +90,25 @@ public void ensureDefaultChannelId() {
         assertTrue(channelId instanceof DefaultChannelId);
     }
 
+    @Test
+    void processIdWithProcessHandleJava9() {
+        Assumptions.assumeTrue(PlatformDependent.javaVersion() >= 9);
```

Comment:
nit: You can use `@EnabledForJreRange` and so get rid of the Assumptions.

Reply:
Unlike assumptions, `@EnabledForJreRange` is not used in netty's codebase. In particular above is used verbatim in many ssl related tests. Which one do you prefer?

Reply:
Let's use `@EnabledForJreRange`... We just migrated everything to juni5 and so should make use of its features